src/styles/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
```

Comment:
Always add `box-sizing: border-box;` on all elements `* {...}`

Reply:
Now for img heigth i should sum padding

Reply:
It is a good practice to use box-sizing: border-box; on all elements, because when your project grows it may become rather hard to track which of your elements have different box-sizing settings and it leads you to tough bugfixing process.

Reply:
Though not all developers use this practice, so you may decide by yourself and leave it like this.

Reply:
Ok thx